scripts/micro_benchmarks.py

```diff
@@ -220,10 +250,40 @@ def __process_arguments(args: list) -> Tuple[list, bool]:
     parser = add_arguments(parser)
     return parser.parse_args(args)
 
+def __getBenchmarkDotNetArguments(
```

Comment:
nit: two lines between global function definitions.

Reply:
this convention is crazy, but ok ;p

Reply:
[PEP 8 -- Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/)

Reply:
You can use linters to help on this too ;)